code/src/components/InputForm.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react'
+
+const InputForm = () =>{
+    const [message, setMessage] = useState("");
+    const THOUGHTS_URL = "https://happy-thoughts-technigo.herokuapp.com/thoughts";
+    
+    const submit = (event) => {
+        event.preventDefault();
+
+        fetch(THOUGHTS_URL, {
+          method: "POST",
+          headers: { "Content-Type": "application/json" },
+          body: JSON.stringify({ message: message }),
+        })
+            .then(() => {
+            window.location.reload();
+            });
+      };
+
+
+    return (
+    <form onSubmit={submit} className="inputform">
+        <h1>What is making you happy right now ?</h1>
+        <textarea 
+            className="thoughts-textarea"
+            maxlenght="300" 
+            type="text" 
+            onChange={(event) => setMessage(event.target.value)}>
```

Comment:
It's good that you've set a max length, however it's not really known for the user that there is a limit of characters to use. It would perhaps be nice with an indicator showing the amount of characters you have left to write or an error popping up when you try to send the message. Now the message is just not showing up and that might be a bit confusing for the user, they don't know why this happened ☺️